docs/src/appendix/snforge/new.md

```diff
@@ -0,0 +1,23 @@
+# `snforge new`
+
+Create a new Starknet Foundry project at the provided path.
```

Comment:
It would kind of duplicate `--overwrite` docs, but maybe let's say here that the path must be empty.